client/material/tabs.tsx

```diff
@@ -119,10 +151,17 @@ export function Tabs<T>({ children, activeTab, onChange, className }: TabsProps<
         return child
       }
 
+      const hotkeys = [{ keyCode: KEY_NUMBERS[i], ctrlKey: true }]
```

Comment:
Since you added `hotkeys` to TabItemProps, isn't it possible someone sets that now? Shouldn't this forward the existing value if it's set?